cmd/Dockerfile.aarch64

```diff
@@ -31,12 +41,13 @@ RUN \
     chown mongodb:mongodb /data/db
 
 USER mongodb
-
-#copy initialization script for later initialization
-COPY *.js /edgex/mongo/config/
-COPY launch-edgex-mongo.sh /edgex/mongo/config/
-
 #expose Mongodb's port
 EXPOSE 27017
+WORKDIR /edgex-mongo
+COPY --from=builder /edgex-mongo/cmd/edgex-mongo/edgex-mongo /edgex-mongo
```

Comment:
ARM build is currently failing because of this path. Is it a typo? The x86 build only has /edgex-mongo/cmd/edgex-mongo and is passing.

Reply:
yes, thanks